candi/version_map.yml

```diff
@@ -187,3 +159,22 @@ k8s:
       registrar: v2.7.0@sha256:4a4cae5118c4404e35d66059346b7fa0835d7e6319ff45ed73f4bba335cf5183
       snapshotter: v6.2.1@sha256:714aa06ccdd3781f1a76487e2dc7592ece9a12ae9e0b726e4f93d1639129b771
       livenessprobe: v2.9.0@sha256:2b10b24dafdc3ba94a03fc94d9df9941ca9d6a9207b927f5dfd21d59fbe05ba0
+  '1.27':
+    status: available
+    patch: 1
+    bashible: *bashible_k8s_ge_1_23
+    ccm:
+      openstack: v1.27.1
+      yandex: v0.26.1
```

Comment:
Have to add 1.27 support [here](https://github.com/deckhouse/yandex-cloud-controller-manager) and create a tag
Like [this](https://github.com/deckhouse/yandex-cloud-controller-manager/pull/51)

Reply:
[Added](https://github.com/deckhouse/yandex-cloud-controller-manager/pull/56)